From 1796ee69175f8f4cbbb7d42835f54ae1f9c47c94 Mon Sep 17 00:00:00 2001
From: Mattia Rizzolo <mattia@debian.org>
Date: Sat, 21 May 2016 21:55:27 +0000
Subject: skip UnixSocketTest on hurd, as unix sockets with SO_REUSEADDR are
 not supported there

 A little discussion about unix sockets with SO_REUSEADDR can be found on
 https://lists.gnu.org/archive/html/bug-hurd/2016-01/msg00039.html

Forwarded: no
Patch-Name: fix-ftbfs-on-hurd.patch
---
 tornado/test/httpserver_test.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tornado/test/httpserver_test.py b/tornado/test/httpserver_test.py
index 11cb723..3535c56 100644
--- a/tornado/test/httpserver_test.py
+++ b/tornado/test/httpserver_test.py
@@ -581,6 +581,8 @@ class ManualProtocolTest(HandlerBaseTestCase):
 
 @unittest.skipIf(not hasattr(socket, 'AF_UNIX') or sys.platform == 'cygwin',
                  "unix sockets not supported on this platform")
+@unittest.skipIf(sys.platform == 'gnu0',
+                 "unix sockets with SO_REUSEADDR not supported on this platform")
 class UnixSocketTest(AsyncTestCase):
     """HTTPServers can listen on Unix sockets too.
 
